contributor_docs/access.md

```diff
@@ -0,0 +1,34 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means we prioritize the needs of marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is not the first conversation we've had about access and inclusion. We share similar intentions in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
+
+**Please consider this a starting point.** We want to invite more conversations about what access means and how we can prioritize it.
+
+## Kinds of access
+
+Increasing access is not a focus on expanding the raw number of people in the p5.js community. It is a focus on making p5.js available to and approachable for more people who are excluded from the p5.js community (intentionally or not) and people who are excluded from similar tools.
+
+Access here means making p5.js better for:
+
+- People who speak Spanish, Chinese, or other languages besides English
+- People who are Black, Indigenous, or people of Color
+- People who are lesbian, gay, bisexual, trans, or queer
+- People with marginalized genders
+- People who are disabled or sick
+- and other people who are systemically excluded
+
+### Examples
+
+These are examples of efforts we believe to increase access:
+
+- Translating more documentation and other materials into more languages
+- Improving out support for assistive devices (such as screenreaders)
+- Making p5.js error messages more helpful and supportive to developers
```

Comment:
+ Teaching p5.js to groups that are marginalized or excluded within coding and/or the arts.

Reply:
This is really important but I think we could write in a different way... "Teaching p5.js to people that are marginalized or excluded..." could be read as if we were on missionary quest to evangelize people. 
Maybe: 
- Supporting learning and teaching p5.js in communities that have been historically marginalized, excluded, and underrepresented in creative coding and digital arts???  

Just an idea, I think it still needs improvement. @montoyamoraga @lmccart @outofambit  what do you think/help?

Reply:
@lm-n that's a good point! i like @lmccart's because it highlights the act of teaching itself (we have other items that cover documentation and coding work already). hmm, how about something like this?

* Mentoring and supporting learners of p5.js _within_ communities that are historically excluded from and marginalized in creative coding and the digital arts

Reply:
I like "Mentoring and supporting learners of p5.js within communities that are historically excluded from and marginalized in creative coding and the digital arts" 😸 🥰

Reply:
added!